Airplane-tickets-UA.Shared/Announcer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Airplane_tickets_UA.Data;
+
+namespace Airplane_tickets_UA.Shared
+{
+    public class Announcer
+    {
+        public void SendEmailAboutDiscount(ICAO firstAirport, ICAO secondAirport)
+        {
+            //Send Email
```

Comment:
You might be interested in reading the following article: https://docs.microsoft.com/ru-ru/azure/sendgrid-dotnet-how-to-send-email